examples/hosts/app-integration/external-data/src/model-interface/index.ts

```diff
@@ -144,6 +151,16 @@ export interface ITaskList extends IEventProvider<ITaskListEvents> {
 	 */
 	readonly importExternalData: () => Promise<void>;
 
+	/**
+	 * Register the container details and the external resource id with the customer service.
+	 * This will allow the Customer Service to pass on the container information to the Fluid
+	 * Service to send the signal that some new information has come through.
+	 */
+	readonly registerWithCustomerService: (
+		externalTaskListId: string,
+		url: IFluidResolvedUrl | undefined,
```

Comment:
Since the container gives us an `IResolvedUrl` I think we can stick with that type everywhere and skip the dependency on `IFluidResolvedUrl`.